OpenCL_C.txt

```diff
@@ -11031,8 +11034,11 @@ footnote:[{fn-int64-supported}], `ulong`, `half` footnote:[{fn-half-supported}],
   _sub_group_local_id_ (value returned by *get_sub_group_local_id*) to all
   work-items in the subgroup.
 
-  _sub_group_local_id_ must be the same value for all work-items in the
-  subgroup.
+  Behaviour is undefined when the value of _sub_group_local_id_ is not
```

Comment:
Suggest using the spelling "behavior" instead of "behaviour" since the former is more common in the spec (and is used in the description of `work_group_broadcast` above).

Reply:
Indeed! It should be more consistent now.